footprints/templates/search/search.html

```diff
@@ -387,14 +388,15 @@ <h4 class="modal-title">People</h4>
                     <span class="glyphicon glyphicon-download-alt"></span> Export
                 </button>
             {% endif %}
-
             {% flag "gallery_view" %}
             <div class="btn-group ml-2" role="group" aria-label="View Options">
-              <button type="button" class="btn btn-sm btn-white {% if not gallery_view %}active{% endif %}">
+              <button name="gallery_view" type="submit" class="btn btn-sm btn-white {% if not gallery_view %}active{% endif %}" 
```

Comment:
There's some complexity to making these buttons type "submit". The functionality works correctly when clicking Image / List for a given search. However, the `gallery_view` value is not retained when clicking "Next" page or when selecting a facet. (The search form is fairly intricate.)

There's a few ways to tackle this. One way would be to capture form submit and serialize the `gallery_view` value explicitly (in javascript).

Another path would be to transform this button group into radio buttons, which Bootstrap appears to support (https://getbootstrap.com/docs/3.4/javascript/#buttons-checkbox-radio). The form submit would then need to be triggered on click (again, in javascript).

Lmk if you'd like to go through these choices.